package.json

```
@@ -11,6 +11,7 @@
     "watch": "tsc --build --watch tsconfig.packages.json",
     "clean": "tsc --build --clean tsconfig.packages.json",
     "wipe": "npm run clean -ws && rimraf node_modules dist tsconfig.tsbuildinfo && npm i",
+    "build:action": "yarn workspace action pkg",
```

I could be convinced either way that we need this at the root, but we should use npm for it.
```suggestion
    "build:action": "npm run pkg -w action",
```